deployment/terraform/modules/osv/osv_api.tf

```diff
@@ -15,11 +9,18 @@ resource "google_cloud_run_service" "api_backend" {
   template {
     spec {
       containers {
-        image = data.google_container_registry_image.api_backend.image_url
+        image = "us-docker.pkg.dev/cloudrun/container/hello:latest" # Placeholder image.
```

Comment:
Given this is just a template, does it even need to be a valid image?

Reply:
Terraform spits out an error if it's not valid when it's first created.